src/mobster/cmd/generate/oci_image/cyclonedx_wrapper.py

```diff
@@ -0,0 +1,75 @@
+"""
+Module for wrapping CycloneDX SBOMs in a way that is fully supported.
+
+If CycloneDX python lib starts supporting the `formulation` field,
+this can be mostly removed.
+"""
+
+import json
+from dataclasses import dataclass, field
+from typing import Any
+
+from cyclonedx.model.bom import Bom
+from cyclonedx.model.component import Component
+from cyclonedx.output import make_outputter
+from cyclonedx.schema import OutputFormat, SchemaVersion
+
+
+@dataclass
+class CycloneDX1BomWrapper:
+    """
+    Wrapper dataclass that drags the currently unsupported field,
+    `formulation` along the SBOM. Can be removed and rewritten
```

Comment:
Does it mean that the field is in the official schema, but it is not supported by the library?

Reply:
Exactly, yes.